mmcv/transforms/processing.py

```diff
@@ -0,0 +1,1079 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import random
+import warnings
+from typing import Dict, List, Optional, Sequence, Tuple, Union
+
+import numpy as np
+
+import mmcv
+from mmcv.image.geometric import _scale_size
+from mmcv.utils.registry import build_from_cfg
+from .base import BaseTransform
+from .builder import TRANSFORMS
+from .wrappers import Compose
+
+Number = Union[int, float]
+
+
+@TRANSFORMS.register_module()
+class CenterCrop(BaseTransform):
+    """Center crop the image and segmentation mask. If the crop area exceeds
+    the original image and ``pad_mode`` is not None, then pad the original
+    image first.
+
+    Required Key:
+
+    - img
+    - gt_semantic_seg
+
+    Modified Key:
+
+    - img
+    - height
+    - width
+    - gt_semantic_seg
+
+    Added Key:
+
+    - pad_shape
+
+
+    Args:
+        crop_size (Union[int, Tuple[int, int]]):  Expected size after cropping
+            with the format of (h, w). If set to an integer, then cropping
+            height and width are equal to this integer.
+        pad_val (Union[Number, Dict[str, Number]], optional): A dict for
+            padding value. To specify how to set this argument, please see
+            the docstring of class ``Pad``. Defaults to dict(img=0, seg=255).
+        pad_mode (str, optional): Type of padding. Should be: 'constant',
+            'edge', 'reflect' or 'symmetric'. For details, please see the
+            docstring of class ``Pad``. Defaults to 'constant'.
+    """
+
+    def __init__(self,
+                 crop_size: Union[int, Tuple[int, int]],
+                 pad_val: Union[Number, Dict[str,
+                                             Number]] = dict(img=0, seg=255),
+                 pad_mode: Optional[str] = None):
```

Comment:
Maybe add `pad = 'Pad'` for config the pad. Like in mmdet Pad will be inherited for processing mask.